src/main/java/com/google/cloud/spanner/hibernate/SpannerTableExporter.java

```diff
@@ -41,17 +45,36 @@ public SpannerTableExporter(SpannerDialect spannerDialect) {
   }
 
   @Override
-  public String[] getSqlCreateStrings(Table exportable, Metadata metadata) {
-    return new String[]{"test_placeholder"};
+  public String[] getSqlCreateStrings(Table table, Metadata metadata) {
+    /* The current implementation does not support interleaved tables for collections
+     * or UNIQUE constraints/indexes for relationships
+     * */
+    String createTableTemplate = this.spannerDialect.getCreateTableString()
+        + " {0} ({1}) PRIMARY KEY ({2})";
+
+    StringJoiner primaryKeyColNames = new StringJoiner(",");
```

Comment:
Will do for this one. unfortunately for the one below that's just an `Iterator` without the type parameter and it is more trouble than it is worth to make that into a stream.

Reply:
Something like this to get the stream?
```java
Iterable<String> iterable = () -> sourceIterator;
Stream<String> targetStream = StreamSupport.stream(iterable.spliterator(), false);
```

It would be nice to make this method a bit more readable overall.

Reply:
While the .getPrimaryKey().getColumns() returns a nice List that can directly be used as a stream, 
the only way to get the columns of the whole entity gives an old-fashioned `Iterator` which I think is less-readable to force to force to be a stream going from iterator to anonymous function to iterable to spliterator to streamsupport.stream.

Reply:
Fair enough, we can always improve readability later.